dev_docs/tutorials/sample_data.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: kibDevTutorialSampleData
+slug: /kibana-dev-docs/tutorial/sample-data
+title: Add sample data
+summary: Learn how to add sample data to Kibana
+date: 2021-04-26
+tags: ['kibana', 'onboarding', 'dev', 'architecture', 'tutorials']
+---
+
+## Installation from the UI
+
+1) Navigate to the home page
+1) Click "Add data"
```

Comment:
For UI components, we use bold instead of quotes:

1. Click **Add data**.

Reply:
Pushed with c98f1d5